src/engine/DnCWorker.cpp

```diff
@@ -101,7 +101,7 @@ void DnCWorker::popOneSubQueryAndSolve( bool restoreTreeStates )
 
         bool fullSolveNeeded = true; // denotes whether we need to solve the subquery
         if ( restoreTreeStates && smtState )
-            fullSolveNeeded = _engine->restoreSmtState( *smtState );
+            fullSolveNeeded = true; //_engine->restoreSmtState( *smtState );
```

Comment:
Do we want to fully remove this functionality for now? Maybe we want to also purge this option here: https://github.com/NeuralNetworkVerification/Marabou/blob/bdcd044e6d64902d209ac5391285bb31a43662c5/src/configuration/OptionParser.cpp#L148?

Reply:
Good point, I will remove it altogether as it is obsolete now.